Sources/Mantis/CropView/CropViewModel.swift

```diff
@@ -8,7 +8,7 @@
 
 import UIKit
 
-enum ImageRotationType: CGFloat {
+public enum ImageRotationType: CGFloat {
```

Comment:
Why do we need this to be public now?

Reply:
I think I forgot deleting some test code. I will remove `public`

Reply:
Fixed